src/components/ContactSalesForm/ContactSalesForm.module.scss

```diff
@@ -0,0 +1,108 @@
+.contactSalesForm {
+  width: 100%;
+
+  &.failed {
+    --state-color: var(--error);
+  }
+
+  &.success {
+    --state-color: var(--success);
+  }
+
+  &.loading::before {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background: adjust-color($color: c(brand-water), $saturation: -30%, $alpha: -0.2);
+    content: '';
+  }
+
+  .state {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    border-radius: 1rem;
+    padding: 0 2rem;
+    width: 100%;
+
+    text-align: center;
+
+    @media screen and (max-width: 640px) {
```

Comment:
We need to fix this and other media queries after https://github.com/fingerprintjs/fingerprintjs.com/pull/40/files is merged

Reply:
Should I add a `// TODO`?

Reply:
You can fix now